### PR TITLE
Change usages of ToLower into ToLowerInvariant

### PR DESCRIPTION
**Pull request by ersagunkuruca**
This fixes a very hard-to-spot bug where Web3Auth & OpenLogin started to give various configuration errors on some specific devices. Here is where we reported it: https://web3auth.io/community/t/invalid-constructor-params-invalid-environment-settings/6643

It turns out all those devices were Turkish, and the character "ı" (or "dotless i" as non-Turkish people would say) was the reason. 

You see, in Turkish, lowercasing/uppercasing rules are different, because we have a distinction between "I" and "İ" where "I".ToLower() == "ı" and "İ".ToLower() == "i" which makes sense in Turkish, and "I".ToLower() == "i" makes sense for any other language, but obviously these two rules don't mix well. And ToLower() uses the current culture information of the device. Which can make network.ToString().ToLower() into "maınnet" and buildEnv.ToString() into "productıon" "stagıng" or "testıng".

To sum up: ToLower uses current device's region settings for lowercasing a string, which can be different for some Turkic languages. ToLowerInvariant is what should be used where this behavior doesn't make sense.

### CODE REVIEW

**Comment by chaitanyapotti** [comment]
Thanks for this PR

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Comment by ersagunkuruca** [comment]
Thanks for the quick approval. Also sorry about the sloppy commit, it's because I edited the file on Github, my bad.
